spec/index.bs

```diff
@@ -1545,6 +1545,14 @@ interface GPUMapMode {
 
   **Returns:** {{ArrayBuffer}}
 
+  1. If |offset| is 0:
```

Comment:
I'm not sure about this, because setting an offset to 0 could also be a programmer error because they thought they mapped the buffer starting from 0. Maybe we could have this only if offset isn't specified?

Reply:
We default it to 0 in case the user doesn't specify it (https://gpuweb.github.io/gpuweb/#ref-for-dom-gpubuffer-getmappedrange). We should, maybe, change the argument to nullable instead of defaulting to 0.

Reply:
Yep that would make sense, but I'd like spec editors to weigh on this issue as well.

Reply:
yes, we need to make it nullable here